config/routes.rb

```diff
@@ -1,13 +1,12 @@
 Rails.application.routes.draw do
   devise_for :users
-    namespace :api do
-      namespace :v1 do
-        resources :games, only: [:index, :create, :show] do
-
-          resources :reviews, only: [:new, :create, :index]
-
-        end
+  namespace :api do
+    namespace :v1 do
+      resources :games, only: [:index, :create, :show] do
+        resources :reviews, only: [:new, :create, :index]
+      end
     end
   end
-
+  root 'games#index'
+  get '*path', to: 'games#index'
```

Comment:
You should no longer have a catch all path e.g. *path

Instead, please add explicit paths: 
get '/games', to: 'games#index'
get '/games/:id', to: 'games#index'